src/app/components/popover/index.html

```diff
@@ -48,14 +48,30 @@ <h3>Positioning</h3>
         </sky-docs-visual-example-image>
       </sky-docs-visual-example>
       <h3>Trigger</h3>
-        <div class="sky-text-block">
-          <p>
-            To ensure usability on touch devices, trigger user-invoked popovers on <sky-code>click</sky-code> actions rather than <sky-code>mouseenter</sky-code> actions.
-          </p>
-          <p>
-            You can also invoke popovers programmatically. For example, you can programmatically invoke a popover when users visit a page for the first time to highlight an important interface element that they might overlook. However, don't invoke multiple popovers on a page because this reduces their effectiveness.
-          </p>
-        </div>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>
+              To ensure usability on touch devices, trigger user-invoked popovers on <sky-code>click</sky-code> actions rather than <sky-code>mouseenter</sky-code> actions.
+            </p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <video controls>
+              <source src="~/assets/img/guidelines/popover/popover-click.mp4" type="video/mp4" />
+            </video>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>
+              You can also invoke popovers programmatically. For example, you can programmatically invoke a popover when users visit a page for the first time to highlight an important interface element that they might overlook. However, don't invoke multiple popovers on a page because this reduces their effectiveness.
+            </p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <video class="bordered" controls>
+              <source src="~/assets/img/guidelines/popover/popover-auto.mp4" type="video/mp4" />
```

Comment:
Is it intentional that this video has the `bordered` class but the previous one does not? When I serve locally, the previous video has a clearly defined, light-grey border that looks the video is smaller than it's container and with a drop shadow within that container. With this video, the image comes right to the edge of its container, with now light-grey border and now drop shadow. Are these differences intentional?

Reply:
It is intentional, but looking at it we can probably update the first video to not use the shadowed border (which is part of the UI in the video). The intent was for it to look more like a modal, but it's not really needed in this context.